app/src/main/res/layout/topic_fragment.xml

```diff
@@ -1,18 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout
-  xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
+
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent">
-    <TextView
-      android:id="@+id/dummy_text_view"
-      android:layout_width="wrap_content"
+
+    <com.google.android.material.appbar.AppBarLayout
+      android:id="@+id/appBarLayout"
```

Comment:
Layout IDs should be defined using ``snake_case``. Also, let's make this contextualized based on the container since Android IDs are global, and 'appBarLayout' isn't clear which layout it corresponds to when referenced in Kotlin.

Suggest: ``topic_app_bar_layout``.

Reply:
resolved